### PR TITLE
Add test page for javascript log output. JB#55880 OMP#JOLLA-435

### PR DESCRIPTION
**Pull request by llewelld**
This is just a really simple test page. It calls various console logging methods on load.

To be deployed to:
https://browser.sailfishos.org/tests/testlogging.html

### CODE REVIEW

File: tests/manual/testlogging.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+	<head>
```
Comment (rainemak) [inline tests/manual/testlogging.html:3]: Use spaces instead of tabs.

File: tests/manual/testlogging.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+				console.log("This is a debug log");
+				console.warn("This is a warning log");
+				console.error("This is an error log");
+				console.assert(true, "This is an assert log");
+				throw new Error("This is throwing an error");
+			}
+		</script>
+	</head>
+	<body onLoad="init()">
+		<p/> This is a console logging test.
```
Comment (rainemak) [inline tests/manual/testlogging.html:17]:
```suggestion
		<p>This is a console logging test.</p>
```

**Review by rainemak — APPROVED** [review]
Minor remarks but approving already. Does what it says. :)

File: tests/manual/testlogging.html (hunk 1 above)
Comment (llewelld) [reply tests/manual/testlogging.html]: Thanks, fixed!

File: tests/manual/testlogging.html (hunk 2 above)
Comment (llewelld) [reply tests/manual/testlogging.html]: Fixed throughout.

**Comment by llewelld** [comment]
I added an extra test for trace calls with a deeper stack.

**Comment by rainemak** [comment]
Nice! Nothing to add.

**Review by okodron — APPROVED** [review]

**Pull request merged**